task-4/task-4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace task_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, user!");
+            FullEquationCalculator();
+            Console.WriteLine("/// type to exit");
+            Console.ReadKey();
+        }
+
+        static void FullEquationCalculator() 
+        {
+            double answer = smallCalc(3, 5) * smallCalc(3, 7) * smallCalc(5, 5);
+            Console.WriteLine($"Your answer is {Math.Round(answer, 2)}");
+        }
```

Comment:
Лучше если название метода начинается с глагола (метод - это действие!). Слово _Equation_ в имени метода неудачное и сбивает с толку. _Equation_ - это уравнение. Но метод не решает уравнение а вычисляет значение выражения. 
Еще одно замечание - не следует нагружать метод двумя разными задачами. Хорошим стилем считается, если метод делает что-то одно. У вас он и вычисляет, и выводит данные на консоль. Лучше оставить методу вычисление с возвратом вычисленного значения.А вывод результата сделать в `Main`.
```suggestion
        static double CalculateFullExpression() 
        {
            return CalculateSmallExpression(3, 5) * CalculateSmallExpression(3, 7) * CalculateSmallExpression(5, 5);
        }
```